docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+For more conceptual information about identities and permissions in a Fabric-based blockchain network, see [Identity](identity/identity.html) and [Membership](membership/membership.html).
+
+For a look at how to use a CA to generate an admin identity and MSP, check out [Enroll Org1's Admin](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin).
+
+To see how to use the TLS CA to generate the certificates and local MSP of a node, check out [Setup Org1's Peers](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#setup-org1-s-peers).
+
+## Step five: Deploy nodes
+
+Once you have gathered all of the certificates and MSPs you need, you're almost ready to create a node. As discussed above, there are a number of valid ways to deploy nodes.
+
+### Create a peer
+
+Before you can create a peer, you will need to customize the configuration file for the peer. In Fabric, this file is known as `core.yaml`. You can find a sample `core.yaml` configuration file [in the sampleconfig directory of Hyperledger Fabric](https://github.com/hyperledger/fabric/blob/master/sampleconfig/core.yaml).
+
+As you can see in the file, there are quite a number of parameters you either have the option to set or will need to set for your node to work properly. In general, if you do not have the need to change a tuning value, leave it alone. You will, however, likely need to adjust the various addresses, specify the database type you want to use, as well as to specify where the MSP for the node is located.
+
+You have two main options for tuning your configuration.
+
+1. Edit the YAML file bundled with the binaries.
+2. Use environment variable overrides when deploying (if using Docker).
+
+The former option has the advantage of persisting your changes whenever you bring down and bring back up the node. The downside is that you will have to port the options you customized to the new YAML when upgrading to a new binary version (you should use the latest YAML when upgrading to a new version). By using a file like `./env<name of node>.list` that contains your environment variables, it is easy to port your changes to a new YAML.
+
```

Comment:
Kubernetes 'Services' also maintain the cluster network connections for configuration preservation no matter how many times pods are restarted.  This is actually a huge advantage.  Just included this for context, not expecting you to add to docs.